packages/calcite-components/src/components/rating/rating.scss

```diff
@@ -4,6 +4,11 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-rating-spacing-unit: The amount of left and right margin spacing between each rating star.
+ * @prop --calcite-rating-hover-selected-color: Specifies the component's color when hovered and selected.
+ * @prop --calcite-rating-idle-unselected-color: Specifies the component's color when idle and unselected.
```

Comment:
Can this just be `--calcite-rating-color`? Or does it need to specify idle and unselected?

Reply:
I updated this token to `--calcite-rating-color`.